### PR TITLE
Chaos/model args

### PR DESCRIPTION
**Pull request by 0xcha05**
added openai model to args, modified args code, updated readme

### CODE REVIEW

**Comment by 0xcha05** [comment]
oops, this commit also came in! my bad!
https://github.com/Torantulino/Auto-GPT/pull/18

File: README.md [hunk 1]
```diff
@@ -1,28 +1,37 @@
...
-- [Installation](#-Installation)
-- [Usage](#-Usage)
-- [Limitations](#-Limitations)
-- [Disclaimer](#-Disclaimer)
-- [Connect with Us on Twitter ](#-connect-with-us-on-twitter)
-
+- [Auto-GPT: An Autonomous GPT-4 Experiment](#auto-gpt-an-autonomous-gpt-4-experiment)
+  - [Demo (30/03/2023):](#demo-30032023)
+  - [💖 Help Fund Auto-GPT's Development](#-help-fund-auto-gpts-development)
+  - [Table of Contents](#table-of-contents)
```
Comment (Torantulino) [inline README.md]:
```suggestion
- [💖 Help Fund Auto-GPT's Development](#-help-fund-auto-gpts-development)
```

Comment (Torantulino) [reply README.md]: Some of those don't make sense as elements in the contents table.

File: scripts/agent_manager.py
Comment (Torantulino) [inline scripts/agent_manager.py:1]: No changes have been made here other than formatting.

Please selectively commit files with genuine changes in future pull requests, as it makes it so much easier for me to review 😉

File: scripts/config.py [hunk 2]
```diff
@@ -19,9 +19,13 @@ class Config(metaclass=Singleton):
     def __init__(self):
         self.continuous_mode = False
         self.speak_mode = False
+        self.openai_model = "4"
```
Comment (Torantulino) [inline scripts/config.py]:
```suggestion
        self.openai_model = "gpt-4"
```

File: scripts/execute_code.py [hunk 3]
```diff
@@ -18,20 +19,22 @@ def execute_python_file(file):
...
+            },
+            working_dir="/workspace",
             stderr=True,
             stdout=True,
             detach=True,
         )
 
         output = container.wait()
-        logs = container.logs().decode('utf-8')
+        logs = container.logs().decode("utf-8")
```
Comment (Torantulino) [inline scripts/execute_code.py:35]: What's the reason behind this change @0xcha05?

File: scripts/execute_code.py
Comment (Torantulino) [inline scripts/execute_code.py:1]: Formatting.

**Review by Torantulino — CHANGES_REQUESTED** [review]
Thank you so much for your pull request!

I want to implement your changes this as soon as possible, but in order to make this a smoother process, can you please amend the pull-request to contain only non-formatting changes .

This would make safely reviewing this code for everyone to use a much more manageable task,  as currently there are over 400 lines of code to review!

You can [easily commit individual lines to Git using VSCode](https://stackoverflow.com/questions/34730585/how-can-i-commit-some-changes-to-a-file-but-not-others-in-vscode). I always commit like this, that way each commit can contain a single, clearly defined change.

If you're unsure where to start, I'd create a fresh fork and copy and paste the specific lines you changed over to the new version, committing each defined change as you go.

I hope you find this to be reasonable, I've very open to criticism or feedback :)

**Comment by 0xcha05** [comment]
I think what you said is reasonable, I have an autoformatter on save in vscode.so I formatted all the files to keep it consistent.  I think that answers most of the comments in here. 
Its usually a good idea to format to make it more readable, so i suggest we keep the formating. but let me know if you don't want those so I can revert it.

File: scripts/execute_code.py (hunk 3 above)
Comment (0xcha05) [reply scripts/execute_code.py:35]: autoformat in vscode

File: README.md (hunk 1 above)
Comment (0xcha05) [reply README.md]: autoformat in vscode, does it on save. wanted to keep it consistent through out.

File: scripts/agent_manager.py
Comment (0xcha05) [reply scripts/agent_manager.py:1]: I understand that it's a little harder when people push in formatting changes also.

**Comment by 0xcha05** [comment]
I'm using the prettier vs code extension for formatting btw. please give it a try, makes it more readable.

**Comment by Torantulino** [comment]
@0xcha05, Would you mind submitting a fresh pull-request?

The process of managing this pull request taught me a lot, and there are now new systems in place!
I think I may have broken your changes in this request when I was merging, so could you please submit your changes again? 

Thank you for your patience.

**Comment by 0xcha05** [comment]
sure, i'll open a fresh PR.
